pkg/devicemapper/devmapper_log.go

```diff
@@ -5,17 +5,31 @@ package devicemapper
 import "C"
 
 import (
+	"fmt"
 	"strings"
+
+	"github.com/Sirupsen/logrus"
 )
 
+// dmLogLevel gates what level of libdm's debugging output will be forwarded
+// through logrus. libdm uses logging *very* liberally and most people using
+// --debug [or Docker] aren't going to be debugging libdm bugs. You can enable
+// this by setting --storage-opt="dm.log_level=X".
+var dmLogLevel = LogLevelErr
```

Comment:
So this changes default from fatal to level Err? When I had tested last time, even with this level libdm was spewing tons of messages. And it was made worse because we try device removal in a loop. Most of the time these messages are really not worth it. 

How about keeping the current defaults and providing a knob to change that default.